test/helper/mockHealth.ts

```diff
@@ -0,0 +1,87 @@
+import { MockAgent, setGlobalDispatcher, getGlobalDispatcher, Dispatcher } from 'undici'
+import env from '../../src/env'
+import { responses as healthResponses } from './healthHelper'
+
+const okResponse = healthResponses.ok
+const ipfsDown = healthResponses.ipfsDown
+const substrateDown = healthResponses.substrateDown
+
+export async function withOkMock() {
+  let originalDispatcher: Dispatcher
+  let mockAgent: MockAgent
+  beforeEach(async function () {
+    originalDispatcher = getGlobalDispatcher()
+    mockAgent = new MockAgent()
+    setGlobalDispatcher(mockAgent)
+    const mockIdentity = mockAgent.get(`http://${env.IPFS_HOST}:${env.IPFS_PORT}`)
+    mockIdentity
+      .intercept({
+        path: `http://${env.IPFS_HOST}:${env.IPFS_PORT}/api/v0/version`,
```

Comment:
I think there's duplicate information here. Should this not be `/api/v0/version` because the host is already set on ln 16